packages/mpv.rb

```diff
@@ -3,23 +3,22 @@
 class Mpv < Package
   description 'Video player based on MPlayer/mplayer2'
   homepage 'https://mpv.io/'
-  version '0.33.1'
+  @_ver = '0.33.1'
+  version "#{@_ver}-1"
   license 'LGPL-2.1+, GPL-2+, BSD, ISC and GPL-3+'
   compatibility 'all'
```

Comment:
I think we can drop i686 support since sommelier doesn't work and this project requires it.

Reply:
Should we leave it as compatible and just not add binaries?

I don't know that it won't build... but let the user deal with that.

As we move to deprecate i686, it might make sense to leave i686 as a specifically excluded outlier, rather than manually change packages to everything BUT i686.

Reply:
I agree with this line of thinking.  It would be really nice if we could get sommelier working in i686 again.  Might have to go way back in the repo tree to find a commit that works, however.